codegenerator/cli/src/config_parsing/human_config.rs

```diff
@@ -361,7 +376,8 @@ pub mod evm {
         )]
         pub url: SingleOrList<String>,
         #[serde(skip_serializing_if = "Option::is_none")]
-        pub unstable__sync_config: Option<SyncConfigUnstable>,
+        #[schemars(description = "Config options for RPC syncing")]
+        pub sync_config: Option<RpcSyncConfig>,
```

Comment:
Feels like it should be flattend in the `RpcConfig`

Reply:
Ok sure we can do that 👍🏼